src/scripts/itembynamedetailpage.js

```diff
@@ -275,6 +275,12 @@ function loadItems(element, item, type, query, listOptions) {
             element.querySelector('a').classList.add('hide');
         }
 
+        // If results are empty, hide the section
+        if (!result.Items?.length) {
+            element.classList.add('hide');
+            return;
+        }
+
```

Comment:
Should we also skip the block above?
Or, on the other hand, shouldn't we clear `itemsContainer.innerHTML` on reloading if the metadata has changed?